config/prod/prow/jobs/custom-jobs.yaml

```diff
@@ -212,13 +212,20 @@ periodics:
   agent: kubernetes
   decorate: true
   cluster: "build-knative"
+  extra_refs:
+  - org: knative
+    repo: test-infra
+    base_ref: master
+    path_alias: knative.dev/test-infra
   spec:
     containers:
-    - image: gcr.io/knative-tests/test-infra/prow-auto-bumper:latest
+    - image: gcr.io/knative-tests/test-infra/prow-tests-go114:stable
       imagePullPolicy: Always
       command:
-      - "/prow-auto-bumper"
+      - "go"
       args:
+      - "run"
+      - "./tools/prow-auto-bumper/."
```

Comment:
It's the first time I've seen path like this, but seems it does work 😅 

/lgtm